charts/artifact-hub/templates/hub_deployment.yaml

```diff
@@ -26,6 +25,9 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
     {{- end }}
+      {{- if .Release.IsInstall }}
+      serviceAccountName: hub
+      {{- end }}
```

Comment:
I'm not sure I understand why this SA should only be used when installing.

Reply:
Only when installing we check if the db-migrator job has completed. After the first upgrade, the hub should never talk anymore to the k8s api, so it doesn't need the service account.

Reply:
I understand.

Reply:
Thanks for your help @makkes. I'll merge this now then. Happy to reopen the discussion and make the necessary changes if you can think of any issue at some point.